src/pages/api/coverage.ts

```diff
@@ -1,11 +1,15 @@
 import { NextApiRequest, NextApiResponse } from 'next';
 
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-const globalAny : any = global;
+const globalAny: any = global;
 
-export default (req : NextApiRequest, res : NextApiResponse) : void => {
-    // TODO: Disable this in production
-    res.status(200).json({
-        coverage: globalAny.__coverage__ || null,
-    });
-};
\ No newline at end of file
+export default (req: NextApiRequest, res: NextApiResponse): void => {
+    if (process.env.NODE_ENV === 'production' && !process.env.NEXT_PUBLIC_CYPRESS) {
```

Comment:
The name of the variable needs to be changed here as well as a consequence of the change in 7fba7130dc4a80533dc38959d7960f6bd9d8bba3.